readme.md

```diff
@@ -9,16 +10,13 @@ Add Google Calendar events to your WordPress site in minutes. Beautiful calendar
 
 Premium add-ons available at [simplecalendar.io](https://simplecalendar.io)
 
-## Run Tailwind 
-To Run Tailwind Watch Use This Command .
-npm run tailwind .
-
-
-
 ## Support
 
 GitHub is for *bug reports and contributions only*. If you have a support question or a request for a customization this is not the right place to post it. Use the [WordPress.org support forums](https://wordpress.org/support/plugin/google-calendar-events) instead.
 
 ## Contributing
 
 Read further on how to [contribute to this project](https://github.com/Xtendify/Simple-Calendar/blob/main/contributing.md).
+
+To Run Tailwind Watch Use This Command .
```

Comment:
This can be moved to https://github.com/Xtendify/Simple-Calendar/blob/main/contributing.md

Reply:
Done